README.md

```diff
@@ -20,10 +22,13 @@ The OAuth2 authentication of the API for transmart-server is managed by [transma
 
 As an user interface for transmart-server there is a frontend application [transmartApp](transmartApp) built as a "web" profiled Grails platform plugin. There is also a modern cohort selector for i2b2 tranSMART named [Glowing Bear](https://github.com/thehyve/glowing-bear) that can used both with transmart-server and transmart-api-server.
 
+The database schema is described in the [data model documentation](docs/data-model.md).
+The schema is created using Liquibase at application startup, when it is configured to do so
+(see [Setup database](docs#2-setup-database)).
```

Comment:
Maybe also usefull to mention that both ways should work for postgres and oracle (not sure if the liquibase works with oracle though, it should, right?)

Reply:
I created and tested it for `postgresql` and `h2`. I doubt if it works for Oracle, because of the `serial` data type. Should not be hard to make it work, but we would need time to test. I will add a note about it.